nerfums-app/src/app/auth/register-user/register-user.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, OnInit } from '@angular/core';
+import { ThrowStmt } from '@angular/compiler';
+import { FormControl, Validators, FormGroup, FormBuilder } from '@angular/forms';
+import { NerfumsService } from '../../nerfums.service';
+import { stringify } from 'querystring';
+import { User } from 'src/model/User';
+
+@Component({
+  selector: 'app-register-user',
+  templateUrl: './register-user.component.html',
+  styleUrls: ['./register-user.component.css']
+})
+export class RegisterUserComponent implements OnInit {
+  submitted: Boolean = false;
+  
+  private full_name: String = "";
```

Comment:
any reason you're using `String` and `Boolean` instead of `string` and `boolean`? The upper case variant is the object type, vs primitive type, which should be favored.

Reply:
no, just habbit from java, let me look into the differences